omod/src/main/compass/sass/providermanagement/bootstrap.scss

```diff
@@ -0,0 +1,6760 @@
+/*!
```

Comment:
Okay... looks like there is no compass stuff in uicommons right now, so not worth trying to set it all up at this point.

Let's keep bootstrap  in coreapps then, but let's slide it up to the top-level sass directory instead of the providermanagement directory since others could use it.